gravitee-apim-portal-webui-next/src/scss/theme/variables.scss

```
@@ -92,16 +95,16 @@ $button-text-container-shape: var(--gio-app-button-text-container-shape, $contai
 
 // -- Search Bar --
 $search-bar-container-shape: var(--gio-app-search-bar-container-shape, $container-shape);
-$search-bar-border-color: var(--gio-app-search-bar-outline-color);
+$search-bar-border-color: var(--gio-app-search-bar-outline-color, $border-color);
 $search-bar-icon-color: var(--gio-app-search-bar-icon-color, $primary-main-color);
 $search-bar-input-text-color: var(--gio-app-search-bar-input-text-color);
 
 // -- Table --
-$table-border-color: var(--gio-app-table-outline-color, color-mix(in srgb, $secondary-main-color 35%, transparent));
+$table-border-color: var(--gio-app-table-outline-color, $border-color);
 $table-border-radius: var(--gio-app-table-container-shape, $container-shape);
-$table-header-background-color: var(--gio-app-table-header-container-color, $secondary-main-color);
+$table-header-background-color: none;
```

```suggestion
$table-header-background-color: var(--gio-app-table-header-container-color, none);
```
WDYT so that the user could still use a token to override the header background color?

---

done